drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -643,89 +720,26 @@ The trace option data MUST be 4-octet aligned:
-
-                  <t hangText="Bit 1">When set indicates presence of
-                  ingress_if_id and egress_if_id (short format) in the node
-                  data.</t>
-
-                  <t hangText="Bit 2">When set indicates presence of timestamp
-                  seconds in the node data.</t>
-
-                  <t hangText="Bit 3">When set indicates presence of timestamp
-                  subseconds in the node data.</t>
-
-                  <t hangText="Bit 4">When set indicates presence of transit
-                  delay in the node data.</t>
-
-                  <t hangText="Bit 5">When set indicates presence of app_data
-                  (short format) in the node data.</t>
-
-                  <t hangText="Bit 6">When set indicates presence of queue
-                  depth in the node data.</t>
-
-                  <t hangText="Bit 7">When set indicates presence of variable
-                  length Opaque State Snapshot field.</t>
-
-                  <t hangText="Bit 8">When set indicates presence of Hop_Lim
-                  and node_id in wide format in the node data.</t>
-
-                  <t hangText="Bit 9">When set indicates presence of
-                  ingress_if_id and egress_if_id in wide format in the node
-                  data.</t>
-
-                  <t hangText="Bit 10">When set indicates presence of app_data
-                  wide in the node data.</t>
-
-                  <t hangText="Bit 11">When set indicates presence of the
-                  Checksum Complement node data.</t>
-
-                  <t hangText="Bit 12-15">Undefined. An IOAM encapsulating
-                  node must set the value of each of these bits to 0. If an
-                  IOAM transit node receives a packet with one or more of
-                  these bits set to 1, it must either: <list style="numbers">
-                      <t>Add corresponding node data filled with the reserved
-                      value 0xFFFFFFFF, after the node data fields for the
-                      IOAM-Trace-Type bits defined above, such that the total
-                      node data added by this node in units of 4-octets is
-                      equal to NodeLen, or</t>
-
-                      <t>Not add any node data fields to the packet, even for
-                      the IOAM-Trace-Type bits defined above.</t>
-                    </list></t>
-                </list></t>
-
-              <t hangText=" "><xref target="trace-node-data-element"/>
-              describes the IOAM data types and their formats. Within an
-              in-situ OAM domain possible combinations of these bits making
-              the IOAM-Trace-Type can be restricted by configuration
-              knobs.</t>
+              <t anchor="IOAMTraceType" hangText="Namespace-ID:">16-bit
+              identifier of an IOAM namespace. The Namespace-ID value of
+              0x0000 is considered as default and known to all the nodes
```

Comment:
s/is considered as default and known/is defined as the default value and MUST be known/